lib/controllers/help.js

```diff
@@ -2,17 +2,21 @@ const helpText = `Minimum viable CLI for the MythX security analysis platform.
 
 USAGE:
 
-$ sabre [options] <solidity_file> [contract_name]
+$ sabre
 
+COMMANDS:
```

Comment:
The `list`, `status` and `report` commands are still missing.

Reply:
Resolved